string.c

```diff
@@ -11289,21 +11253,12 @@ rb_str_unicode_normalize_bang(int argc, VALUE *argv, VALUE str)
 }
 
 /*  call-seq:
- *    str.unicode_normalized?(form=:nfc)
+ *   unicode_normalized?(form = :nfc) -> true or false
  *
- *  Checks whether +str+ is in Unicode normalization form +form+,
+ *  Returns whether +self+ is in Unicode normalization form +form+,
  *  which can be any of the four values +:nfc+, +:nfd+, +:nfkc+, or +:nfkd+.
- *  The default is +:nfc+.
- *
- *  If the string is not in a Unicode Encoding, then an Exception is raised.
```

Comment:
I think this should be kept in the documentation, along with the example exception below.

I think we should also keep some of these examples.